app/contributing/style-guide.md

```diff
@@ -56,6 +56,17 @@ All files in `src` are formatted with Prettier. Prose in `app` has not been bulk
   - **Important:** Information that the reader really needs to pay attention to, otherwise things won't work.
 For more information about formatting admonitions see [markdown-rules](/contributing/markdown-rules/#admonitions).
 
+## Content types
+
+At Kong, we use the four following standard content types when we write our documentation:
+
+- [Explanation](https://github.com/Kong/docs.konghq.com/blob/main/docs/templates/explanation-template.md): Documentation that is understanding-oriented because it clarifies and discuss a particular topic.
```

Comment:
```suggestion
- [Explanation](https://github.com/Kong/docs.konghq.com/blob/main/docs/templates/explanation-template.md): Documentation that is understanding-oriented because it clarifies and discusses a particular topic.
```